staging/src/k8s.io/apiserver/pkg/registry/rest/meta.go

```diff
@@ -23,11 +23,19 @@ import (
 	"k8s.io/apimachinery/pkg/util/uuid"
 )
 
+// WipeObjectMetaSystemFields erases fields that are managed by the system on ObjectMeta.
+func WipeObjectMetaSystemFields(meta metav1.Object) {
+	meta.SetCreationTimestamp(metav1.Time{})
+	meta.SetUID("")
+	meta.SetDeletionTimestamp(nil)
```

Comment:
... I think you should have gotten an error if you try to set this on create. Is that true or did I misremember? Is it still true?

Reply:
Which?  Deletion?  I just tried and it was ignored

Reply:
timestamp and grace period both?

Reply:
```
$ cat /tmp/s.yaml 
apiVersion: v1
kind: Service
metadata:
  name: spec-too-much-for-create
  uid: 1670ac55-2f7c-3f03-9264-b1328e7bcd9f
  creationTimestamp: "2021-10-30T23:15:42Z"
  deletionTimestamp: "2021-11-30T23:15:42Z"
  selfLink: "foo"
  deletionGracePeriodSeconds: 33
  resourceVersion: "abc123"
  generation: 66
spec:
  ports:
  - port: 80
    targetPort: 9376
  selector:
    app: hostnames
  type: ClusterIP

$ k create -f /tmp/s.yaml 
service/spec-too-much-for-create created

$ k get svc spec-too-much-for-create -o json | jq .metadata
{
  "creationTimestamp": "2022-06-19T19:04:04Z",
  "generation": 66,
  "name": "spec-too-much-for-create",
  "namespace": "default",
  "resourceVersion": "40152074",
  "uid": "b9875fe5-fee1-447b-beb0-6aaabac2afb7"
}
```

Reply:
Huh, I feel like we should have helped them fix their config rather than "helped" them make an object that's not what they asked for. But too late now.